manim/communitycolors.py

```diff
@@ -1,7 +0,0 @@
-"""The colormap of manim community"""
```

Comment:
it is here on purpose...
until we have something like themes, see the PR where it was created to get the info.

Reply:
https://github.com/ManimCommunity/manim/pull/892

Reply:
A poll on the discord server getting quite a few responses unanimously agreed with me.